### ISSUE TITLE
calendar showing no events

### ISSUE THREAD

**Issue by gramels** [nextcloud/calendar#3823]
### Steps to reproduce
1. login via web browser
2. click calendar
3.

### Expected behaviour
list events (in any view)

### Actual behaviour
0 events, spinning wheel forever

### Calendar app
3.02

IOS & MacOS work sync work fine

### Client configuration
Firefox 95.0.1 (64-bit)
MacOS Safari 15.2

Debian

### Server configuration

**Operating system**: (e.g. Debian 8)

**Web server:** (e.g. Apache, Nginx,...)

**Database:** (e.g. MariaDB, SQLite or PostgreSQL)

**PHP version:** PHP 7.4.25

**Nextcloud Version:** 22.2.3

**Updated from an older installed version or fresh install:**

**List of activated apps:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list

Enabled:
  - accessibility: 1.8.0
  - activity: 2.15.0
  - admin_audit: 1.12.0
  - apporder: 0.13.0
  - bruteforcesettings: 2.3.0
  - calendar: 3.0.2
  - camerarawpreviews: 0.7.15
  - circles: 22.1.1
  - cloud_federation_api: 1.5.0
  - comments: 1.12.0
  - contacts: 4.0.7
  - contactsinteraction: 1.3.0
  - dashboard: 7.2.0
  - dav: 1.19.0
  - federatedfilesharing: 1.12.0
  - federation: 1.12.0
  - files: 1.17.0
  - files_bpm: 0.2.1
  - files_external: 1.13.0
  - files_mindmap: 0.0.25
  - files_pdfviewer: 2.3.1
  - files_rightclick: 1.1.0
  - files_sharing: 1.14.0
  - files_trashbin: 1.12.0
  - files_versions: 1.15.0
  - files_videoplayer: 1.11.0
  - logreader: 2.7.0
  - lookup_server_connector: 1.10.0
  - nextcloud_announcements: 1.11.0
  - notes: 4.2.0
  - notifications: 2.10.1
  - oauth2: 1.10.0
  - password_policy: 1.12.0
  - photos: 1.4.0
  - privacy: 1.6.0
  - provisioning_api: 1.12.0
  - recommendations: 1.1.0
  - serverinfo: 1.12.0
  - settings: 1.4.0
  - sharebymail: 1.12.0
  - support: 1.5.0
  - systemtags: 1.12.0
  - tasks: 0.14.2
  - text: 3.3.0
  - theming: 1.13.0
  - twofactor_backupcodes: 1.11.0
  - twofactor_nextcloud_notification: 3.3.1
  - twofactor_totp: 6.2.0
  - updatenotification: 1.12.0
  - user_status: 1.2.0
  - viewer: 1.6.0
  - weather_status: 1.2.0
  - workflowengine: 2.4.0
Disabled:
  - encryption
  - files_fulltextsearch_tesseract
  - firstrunwizard
  - mail
  - maps
  - spreed
  - survey_client
  - user_ldap

**Comment by gramels** [comment]
seems to be restricted to some users
the user with the issue has lots of calendars and events

**Comment by ChristophWurst** [comment]
check browser logs and nextcloud.log

**Comment by gramels** [comment]
`[Debug] No new notification data received (notifications-main.js, line 2)
[Debug] Polling interval updated to 30000 (notifications-main.js, line 2)
[Error] Unrecognized Content-Security-Policy directive 'worker-src'.

[Warning] No OC found (main.js, line 160)
[Log] JQMIGRATE: Migrate is installed, version 3.3.2 (main.js, line 986)
[Warning] jQuery is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] jQuery is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] jQuery is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] Proxying an event bus of version 1.3.0 with 2.0.0 (notifications-main.js, line 2)
[Debug] Loading theming data for notification bell styling (notifications-main.js, line 2)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Debug] Registering notifications container as a menu (notifications-main.js, line 2)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Debug] Notifications permissions granted (notifications-main.js, line 2)
[Debug] Polling interval updated to 30000 (notifications-main.js, line 2)
[Debug] Started background fetcher as session_keepalive is enabled (notifications-main.js, line 2)
[Warning] Proxying an event bus of version 1.3.0 with 2.1.1 (calendar-main.js, line 2)
[Info] [INFO] calendar:  (calendar-main.js, line 2)
Initial settings:
	- AppVersion: 3.0.2
	- EventLimit: false
	- FirstRun: false
	- ShowWeekNumbers: true
	- ShowTasks: true
	- ShowWeekends: true
	- SkipPopover: true
	- SlotDuration: 00:30:00
	- DefaultReminder: none
	- TalkEnabled: false
	- TasksEnabled: true
	- Timezone: Europe/Zurich
 – {app: "calendar", uid: "gramels"}
[Debug] [DEBUG] @nextcloud/vue: Using version 2.2019c of the timezone database – {uid: "gramels", app: "@nextcloud/vue"} (calendar-main.js, line 2)
[Debug] @nextcloud/calendar-js app is using version 2.2019c of the timezone database (calendar-main.js, line 2)
[Debug] Fallback to locale – "en" (calendar-main.js, line 2)
[Info] [INFO] calendar: Updated moment locale: nextcloud-calendar-fake-locale-de-ch-en – {app: "calendar", uid: "gramels"} (calendar-main.js, line 2)
[Info] session heartbeat polling started (main.js, line 822)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Debug] Got notification data (notifications-main.js, line 2)
[Debug] Polling interval updated to 30000 (notifications-main.js, line 2)
[Debug] [DEBUG] unified-search: Unified Search initialized with the following providers – {0: Object, 1: Object, 2: Object, …} (unified-search.js, line 74)
{0: Object, 1: Object, 2: Object, 3: Object, 4: Object, 5: Object, 6: Object, 7: Object, app: "unified-search", uid: "gramels"}Object
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Warning] $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. (main.js, line 1385)
[Debug] [DEBUG] calendar: Current user principal is /owncloud/remote.php/dav/principals/users/gramels/ – {app: "calendar", uid: "gramels"} (calendar-main.js, line 2)
[Error] TypeError: undefined is not an object (evaluating 'l["{DAV:}getcontenttype"].split') — davCollection.js:353
	qt (calendar-main.js:2:4136457)
	Wt (calendar-main.js:2:4136393)
	$t (calendar-main.js:2:4136043)
	promiseReactionJob
[Debug] No new notification data received (notifications-main.js, line 2)
[Debug] Polling interval updated to 30000 (notifications-main.js, line 2)`

**Comment by gramels** [comment]
the issue appeared after update to 22.0.3 from latest 20.xx

**Comment by gramels** [comment]
server log only showing a lot

[no app in context] Info: Deprecated event type for \OCA\DAV\CardDAV\CardDavBackend::updateCard: Symfony\Component\EventDispatcher\GenericEvent is used

at 2021-12-17T17:25:05+00:00

**Comment by paolosg** [comment]
Possible duplicate #3551

**Comment by bigboss74** [comment]
> Possible duplicate #3551

There is a workaroud available:
https://github.com/nextcloud/calendar/issues/3551#issuecomment-951755247

**Comment by localhost14** [comment]
Same issue here using version 3.0.3.
Workaround #3551 is not working for me.
When i click calendar no appointments or birthdays are showing.

**Comment by gramels** [comment]
still not fixed with 3.0.5
Decks & Circle app disabled

**Comment by gramels** [comment]
fixed with 3.2.2
